manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.locale.en-US.yaml

```
@@ -0,0 +1,58 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
+PackageVersion: 0.12.0
+PackageLocale: en-US
+Publisher: magic-wormhole
```

```suggestion
Publisher: aquacash5
```
Best to have this be the repo owner

---

The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole